_health-care/_js/components/financial-assessment/SpouseInformationSection.jsx

```diff
@@ -17,130 +17,152 @@ class SpouseInformationSection extends React.Component {
     return (
       <div>
         <div>
-          <div className="row">
-            <div className="small-12 columns">
-              <h4>Spouse's Information</h4>
-            </div>
-            <div className="small-3 columns">
-              <ErrorableTextInput
-                  label="First Name"
-                  value={this.props.data.spouseFirstName}
-                  onValueChange={(update) => {this.props.onStateChange('spouseFirstName', update);}}/>
-            </div>
-            <div className="small-3 columns">
-              <ErrorableTextInput
-                  label="Middle Name"
-                  value={this.props.data.spouseMiddleName}
-                  onValueChange={(update) => {this.props.onStateChange('spouseMiddleName', update);}}/>
-            </div>
-            <div className="small-3 columns">
-              <ErrorableTextInput
-                  label="Last Name"
-                  value={this.props.data.spouseLastName}
-                  onValueChange={(update) => {this.props.onStateChange('spouseLastName', update);}}/>
-            </div>
-            <div className="small-3 columns">
-              <ErrorableSelect
-                  label="Suffix"
-                  options={suffixes}
-                  value={this.props.data.spouseSuffix}
-                  onValueChange={(update) => {this.props.onStateChange('spouseSuffix', update);}}/>
-            </div>
-          </div>
+          <h4>Spouse's Information</h4>
+          {this.props.external.receivesVaPension === true &&
+            <p>
+              <strong>
+              You are not required to enter financial information because you
+              indicated you are receiving a VA pension.
+              </strong>
+            </p>
+          }
+          {this.props.external.neverMarried === true &&
+            <p>
+              <strong>
+              You are not required to enter financial information because you
+              indicated you've never had a spouse.
+              </strong>
+            </p>
+          }
+        </div>
+        {this.props.external.neverMarried === false &&
+          <div>
+            <div>
```

Comment:
An extra level of `<div>` seems to be required inside a `{...}` to group the other `<div>`s.